src/pkg/build/assemblers/assembler_sandbox.go

```diff
@@ -28,16 +29,9 @@ func (a *SandboxAssembler) Assemble(b *types.Bundle, path string) (err error) {
 	if _, err := os.Stat(path); err == nil {
 		os.RemoveAll(path)
 	}
-	if err := os.Rename(b.Rootfs(), path); err != nil {
-		if err := copy.Copy(b.Rootfs(), path); err != nil {
-			sylog.Errorf("Sandbox Assemble Failed: %s", err)
-			return err
-		}
-
-		if err := os.RemoveAll(b.Rootfs()); err != nil {
-			sylog.Errorf("Unable to remove Bundle directory: %s", err)
-			return err
-		}
+	cmd := exec.Command("mv", b.Rootfs(), path)
+	if err = cmd.Run(); err != nil {
+		return fmt.Errorf("Sandbox Assemble Failed: %s", err)
```

Comment:
Use `fmt` ... or `sylog` that you're already importing?

Reply:
Here is **returning** `fmt.Errorf()`, meaning we're using the format package to return an `error` type which contains formatted text. We can't return `sylog.Errorf()`, which is a function which prints output and does not return anything.

Reply:
:+1: